content/code-of-conduct.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Code of Conduct"
+date: 2019-10-01T16:32:43-04:00
+draft: false
+---
+
+# Conference Code of Conduct
```

Comment:
I suggest remove this header, because it looks like it's stuttering. (https://deploy-preview-69--opensourcecities.netlify.com/code-of-conduct/). May be update the `title` to be `Conference Code of Conduct` or just leave it as is. I'm fine either way, do whatever you feel best.